newIDE/app/src/AssetStore/ProductPriceTag.js

```diff
@@ -32,7 +34,9 @@ export const formatProductPrice = ({
   const stripePrice = productListingData.prices[0];
   if (!stripePrice) return '';
 
-  return `€ ${i18n
+  const currencyCode = stripePrice.currency === 'USD' ? '$' : '€';
```

Comment:
You can rename this probably to `price` - this is not a `stripePrice` anymore ;)